auditlog/models/rule.py

```diff
@@ -142,6 +142,15 @@ class AuditlogRule(models.Model):
         "Capture Record",
         help="Select this if you want to keep track of Unlink Record",
     )
+    users_to_exclude = fields.Many2many(
```

Comment:
```suggestion
    users_to_exclude_ids = fields.Many2many(
```

Reply:
wondering: wouldn't be better to use groups instead of users?

Reply:
@simahawk if we change this to group, it will change the whole concept.  So, we might as well do it in future PR.

Reply:
Corrections done for users_to_exclude_ids